eli5/sklearn/text.py

```diff
@@ -82,8 +96,12 @@ def _get_weighted_spans_from_union(doc, vec_union, feature_weights):
     named_found_features = []
     for vec_name, vec in vec_union.transformer_list:
         vec_prefix = '{}__'.format(vec_name)
-        feature_fn = lambda x: (
-            x[len(vec_prefix):] if x.startswith(vec_prefix) else None)
+
+        def feature_fn(x):
+            if (not isinstance(x, FormattedFeatureName)
```

Comment:
Do you recall why is FormattedFeatureName needed? I rewrote the if statement into two statements in https://github.com/TeamHG-Memex/eli5/pull/110/, and coverage shows feature_fn never see FormattedFeatureName in our tests.